doc/source/io.rst

```diff
@@ -136,7 +136,11 @@ usecols : array-like or callable, default ``None``
   that correspond to column names provided either by the user in `names` or
   inferred from the document header row(s). For example, a valid array-like
   `usecols` parameter would be ``[0, 1, 2]`` or ``['foo', 'bar', 'baz']``. 
-  Element order is ignored, so ``usecols=[0, 1]`` is the same as ``[1, 0]``.
+  Element order is ignored, so ``usecols=[0, 1]`` is the same as ``[1, 0]``. Element
+  order is ignored, so usecols=[1,0] is the same as [0,1]. To instantiate a
```

Comment:
Could you put these `usecols=[]`  in double back ticks as well? And commas between the elements.

Finally, I have a slight preference for using names like `['foo', 'bar']` instead of `[0, 1]`, even though it's a bit longer.

Reply:
Done.  Please refer to 02715087b61b73c6142abae75b4272018eb07402

I went ahead and encased usecols in double back ticks.

Updated column names per your suggestion.  I was originally trying to keep consistent with the line before **Element order is ignored, so ``usecols=[0, 1]`` is the same as ``[1, 0]``**.